src/services/__tests__/organizationService.test.ts

```diff
@@ -75,14 +81,15 @@ beforeAll(async () => {
   createUserAssociations();
   createSessionAssociations();
 
-  user = (await UserModel.create(userInfo, {
+  userInstance = await UserModel.create(userInfo, {
```

Comment:
what's the reason to reassign this for every test, are these objects mutable ? (otherwise we can just use global constants)

Reply:
This object is assigned once, only here, inside [a `beforeAll()`](https://jestjs.io/docs/en/api#beforeallfn-timeout) method. It's not possible to use a global constant because before assigning its value one must await the resolution of all the above instructions, that are needed to initialize the DB tables.